### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by stevenpaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+```
+$ ec origin getallorigins --accountnumber=ABCDE --mediatypeid=3
+```
+
+In addition to using flags to provide inputs, you may also provide JSON input:
+```
+$ ec origin getallorigins --input='{AccountNumber:ABCD, MediaTypeID=3}'
+```
+
+To view the top-level documentation for the CLI, use the `help` command:
```
Comment (defstream) [inline README.md]: Including that, this is how to view all of the commands will add clarity.

> To view the top-level documentation and available commands for the CLI, use the `help` command:

Comment (stevenpaz) [reply README.md]: Good suggestion @defstream updated

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+This is the official command line interface (CLI) for EdgeCast.
+
+## Getting Started
+
+### Requirements
+- Go 1.17
+
+### Installation
+To install the CLI, use the makefile:
```
Comment (ChanggyuOh) [inline README.md]: I think we should take how edge-compute team did. Basically we should have customers to download installation script and run it in order to install the cli and needed config files as necessary

File: README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+```
+$ make install
+```
+
+### Using the CLI
+Commands follow the following format:
```
Comment (ChanggyuOh) [inline README.md:17]: We haven't got there yet. But we should list all available commands, their sub-commands, and flags...

Comment (stevenpaz) [reply README.md:17]: Yeah, I just didn't want to leave a TODO here.

**Review by wdawg33 — APPROVED** [review]
Looks good to me. Thanks, Steven

**Pull request merged**